javascript/javascript1/week3/whenWillWeBeThere.js

```diff
@@ -0,0 +1,18 @@
+// WHEN WILL WE BE THERE
+
+const travelInformation = {
+    speed: 50,
+    destinationDistance: 432,
+};
+  
+const travelTimeInMinutes = travelInformation.destinationDistance * 60 / travelInformation.speed;
+  
+function convert(num) {
+    const hours = Math.floor(num / 60);
```

Comment:
if you pass the whole object, how would you get the speed and the destination data?
Good reading https://www.educative.io/answers/what-is-object-destructuring-in-javascript